adjust readme for v2 release (#612)
SalesforceCommerceCloud/pwa-kit: packages/internal-lib-build/README.md

> # PWA Kit internal tools

Reviewer: ```suggestion
# PWA Kit Internal Tools
```
^Correct heading style.

> # PWA Kit Internal tools 🛠️

Reviewer: ```suggestion
# PWA Kit Internal Tools
```

Use title case for h1s. For consistency with other README files in our repo, use emoji only on h2s (and put the emoji before the heading text, not after).

A package that contains internal tools used by other libraries in the monorepo.

> It can't run outside the context of this monorepo

Reviewer: ```suggestion
It can't run outside the context of this monorepo.
```

> To see all the available commands

Reviewer: ```suggestion
To see all the available commands:
```

> For see all the available commands

Reviewer 2: ```suggestion
To see all the available commands:
```

```bash
cd packages/internal-lib-build

> ./bin/internal-lib-build --help

Reviewer 3: Minor: I need to add `node` to run the script.
```
node ./bin/internal-lib-build --help
```

```

> To see how it is used inside other packages, see [pwa-kit-dev](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/pwa-kit-dev/package.json#L29)

Reviewer: ```suggestion
To see how the package is used inside other packages, see the [pwa-kit-dev](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/pwa-kit-dev/package.json#L29) package configuration.
```
